### ISSUE TITLE
maven id?

### ISSUE THREAD

**Issue by opoplawski** [fflewddur/tivolibre#13]
Have you thought about a proper maven groupId:artifactId for tivolibre?  I'm contemplating packaging this for Fedora and it would need to have one.  I can default to tivolibre:tivolibre, but figured I'd ask first.

**Comment by fflewddur** [comment]
Good idea, I hadn't thought of that. How about net.straylightlabs.tivolibre:tivolibre?

**Comment by opoplawski** [comment]
Yeah, that's probably the best.

**Comment by mprahl** [comment]
@opoplawski did you end up packaging this?

**Comment by fflewddur** [comment]
@mprahl I've added TivoLibre to Maven Central myself under the name "tivo-libre". You can find it here: http://repo1.maven.org/maven2/net/straylightlabs/tivo-libre/0.7.4/
